source/elements/oneMKL/source/domains/dft/descriptor.rst

```diff
@@ -346,7 +351,84 @@ corresponding configuration values, see :ref:`onemkl_dft_enum_config_param`.
 
 **Descriptor class member table:** :ref:`onemkl_dft_descriptor_member_table`
 
+.. _onemkl_dft_descriptor_set_workspace:
 
+set_workspace
++++++++++++++
+
+Set the workspace for when ``config_param::WORKSPACE_PLACEMENT`` is set to ``config_value::WORKSPACE_EXTERNAL``.
+
+.. rubric:: Description
+
+This function sets the workspace to use when computing DFTs for when an
+external workspace is set. 
+This function may only be called after the descriptor has been committed.
+The provided workspace must be equal to or larger than the required 
```

Comment:
```suggestion
The size of the provided workspace must be equal to or larger than the required 
```

Reply:
Done in [a8cbe88](https://github.com/oneapi-src/oneAPI-spec/pull/509/commits/a8cbe883d58d2f99e6233c6d49701bb36b297de6).